leaderboard/models.py

```diff
@@ -247,22 +424,24 @@ class TestSet(models.Model):
 
     file_format = models.CharField(
         choices=FILE_FORMAT_CHOICES,
-        default=SGML_FILE,
+        default=XML_FILE,
         max_length=4,
     )
 
     src_file = models.FileField(
         blank=True,
         upload_to='testsets',
-        help_text='SGML or text file containing test set source',
+        help_text='SGML, XML or text file containing test set source',
         null=True,
+        validators=[validate_xml_src_testset],
```

Comment:
Validation here will enforce that `file_format` is `XML`. This makes the `file_format` field above redundant.

Should we add a validation method which considers `file_format`?

Reply:
I think this shouldn't enforce the format thanks to:
https://github.com/AppraiseDev/OCELoT/blob/c241adb715b6576422a503ee9f23f90233756a79/leaderboard/models.py#L213-L214
Or I simply didn't get the question :)